operations/patch_util.go

```diff
@@ -349,6 +350,16 @@ func getPatchDisplay(p *patch.Patch, summarize bool, uiHost string) (string, err
 	return out.String(), nil
 }
 
+func getAPIPatchDisplay(apiPatch *restmodel.APIPatch, summarize bool, uiHost string) (string, error) {
+	servicePatchIface, err := apiPatch.ToService()
+	if err != nil {
+		return "", errors.Wrap(err, "can't convert patch to service")
+	}
+	servicePatch := servicePatchIface.(patch.Patch)
```

Comment:
I think it's good form to check ok when converting types, just in case

Reply:
done